website/code_issues.qmd

```diff
@@ -227,4 +227,34 @@ The issue described in the recipe, also applies to the use of other function whi
 
 --------------------------
 
+# Writing Files and Directories to the Home Filespace
+
+## Problem
+
+You are writing by default or in your examples, vignettes or tests in the user's home filespace, including the package directory and `getwd()`.
+
+## Solution 
+
+Omit any default path in your functions and write to `tempdir()` in examples, vignettes or tests.  
+
+
+### Details
+
+<!--adding this is a test for including the CRAN mail "template" for each recipe. This version is a small collapsible text box which I will add for (almost) every Issue. Maybe change the title of the boxes?--> 
+
+::: {.callout-note title="Comment of CRAN maintainers" appearance="simple" icon=false collapse=true}
+
+Please ensure that your functions do not write by default or in your examples/vignettes/tests in the user's home filespace (including the package directory and getwd()). This is not allowed by CRAN policies. Please omit any default path in writing functions. In your examples/vignettes/tests you can write to tempdir().
+
+:::
+
+The [CRAN Repository Policy](https://cran.r-project.org/web/packages/policies.html) states that:
+
+> The code and examples provided in a package should never do anything which might be regarded as malicious or anti-social.
+
+and gives as example:
+
+> Packages should not write in the user’s home filespace (including clipboards), nor anywhere else on the file system apart from the R session’s temporary directory (or during installation in the location pointed to by TMPDIR: and such usage should be cleaned up). Installing into the system’s R installation (e.g., scripts to its bin directory) is not allowed.
```

Comment:
It is known that it is allowed to write to clipboards or other places. But not on CRAN or by default on users' machines. I think the wording should be relaxed a little bit.